src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(){
+    }
+
+    public Circle(String color) {
+        super(color);
+    }
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+        area = Math.PI * radius * radius;
```

Comment:
you shouldn't do this inside of the constructor, you should create an interface with method that will return calculated area of figure and implement such interface in every figure